src/lib/constants.ts

```diff
@@ -107,3 +107,9 @@ export const ACCOUNT_OWNERS_TAB_ID = `tab_${ACCOUNT_OWNERS_ID}`;
 export const ADMINISTRATORS_TAB_ID = `tab_${ADMINISTRATORS_ID}`;
 export const STANDARD_USERS_TAB_ID = `tab_${STANDARD_USERS_ID}`;
 export const NOT_PROVIDED = "Not Provided";
+
+const WHITELISTED_URLS: string[] = [
+    LANDING_URL + HEALTHCHECK
+];
+
+export const isWhitelistedUrl = (url: string): boolean => WHITELISTED_URLS.includes(url);
```

Comment:
I wouldn't add functions to constants file. It's probably better idea to move it to utils and also create some tests for it.